src/components/AppMain/NewCard/NewCard.tsx

```diff
@@ -0,0 +1,61 @@
+import { CardContext } from '../../Context/CardListProvider'
+import styles from './NewCard.module.css'
+import React, { ChangeEvent, useContext } from 'react'
+
+const NewCard = () => {
+	const { setEditingCard, editingCard, setIsAdding, isSecondSide, setIsSecondSide, setCardList } =
```

Comment:
**nit:** you should set `currentSide` as string or enum to have a better control over your UI. So:

```tsx
const NewCard = () => {
  const [card, setCardValues] = useState<Card>({
                 title: '',
		value: '',
    })
  const [currentCardSide, setCardSide] = useState(CardSide.Front)

 const { handleAddCard  } = useContext(CardContext)
}
```